src/Components/Cockpit/NotReady.tsx

```diff
@@ -41,6 +41,20 @@ export const NotReady = ({ enabled }: { enabled: boolean }) => {
             Start socket
           </Button>
         </EmptyStateActions>
+        <EmptyStateActions>
+          <Button
+            variant="link"
+            onClick={(event) => {
+              event.preventDefault();
+              cockpit.jump(
+                '/system/services#/osbuild-composer.socket',
+                cockpit.transport.host
+              );
+            }}
+          >
+            More Info
```

Comment:
Let's maybe make this more descriptive? Like `Open osbuild-composer service in Services` or something?

Reply:
actually, I'm ok with it, looking at the page again.